ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/hypershift/guest/cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest-workflow.yaml

```diff
@@ -3,6 +3,7 @@ workflow:
   steps:
     pre:
     - chain: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision
+    - chain: cucushift-hypershift-extended-enable-qe-catalogsource
```

Comment:
thanks, I wasn't aware that we had been missing this step all along :)
cc @LiangquanLi930

Reply:
We were not missing it, I'm just moving it slightly. The explanation is in the commit:
```
Move cucushift-hypershift-extended-enable-qe-catalogsource out of hypershift-provision chain
    This commit moves cucushift-hypershift-extended-enable-qe-catalogsource
    out of cucushift-installer-rehearse-aws-ipi-ovn-hypershift-provision-chain and
    lists the chain explicitly in workflows where it's required.
    Installing qe-catalogsource requires Nodes to be ready and this might
    not be true in some cases like when we install Cilium (in
    cucushift-installer-rehearse-aws-ipi-cilium-hypershift-guest-workflow).
    The Nodes are only ready after installing Cilium and the
    qe-catalogsource can be installed only after that.
```